### ISSUE TITLE
Physics freezes after all child views finished movement

### ISSUE THREAD

**Issue by Evgeniy-Zh** [Jawnnypoo/PhysicsLayout#32]
Hello, I'm looking for a way to change this behavior:

**Comment by Evgeniy-Zh** [comment]
https://drive.google.com/open?id=1G7nU5kuF6RSRT_W_I9iBx0F3fI_-63VQ

**Comment by Jawnnypoo** [comment]
Hmm, not too sure about this one. Could be something with this library, or something strange with the gravity sensor API. I haven't see this happen on my devices, but open to suggestions/PRs fixing it!

**Comment by MikeSimpson** [comment]
I encountered this issue too. It seems to be due to the physics engine putting objects to sleep once the land and stop moving. When gravity is fixed this makes sense, as a physics update on it won't do anything. But when gravity is not constant it results in odd behavior. I added bodyDef.allowSleep = false when configuring the PhysicsConfig for a body. I couldn't get it to work globally unfortunately, setting isAllowSleep or isSleepingAllowed on the World didn't work.

**Comment by Evgeniy-Zh** [comment]
> I encountered this issue too. It seems to be due to the physics engine putting objects to sleep once the land and stop moving. When gravity is fixed this makes sense, as a physics update on it won't do anything. But when gravity is not constant it results in odd behavior. I added bodyDef.allowSleep = false when configuring the PhysicsConfig for a body. I couldn't get it to work globally unfortunately, setting isAllowSleep or isSleepingAllowed on the World didn't work.

Yes, it worked! Thanks!
